bin/check-redis-memory.rb

```diff
@@ -58,6 +58,12 @@ class RedisChecks < Sensu::Plugin::Check::CLI
          description: 'Critical instead of warning on connection failure',
          default: false
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
+         default: 'unknown',
+         in: %w(unknown warning critical ok)
```

Comment:
I question the value of `ok` as an option...A connection failure is clearly not OK...